packages/combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -944,14 +926,6 @@ export const ComboBoxMixin = (subclass) =>
       this.filter = '';
     }
 
-    /**
-     * @return {string}
-     * @protected
-     */
-    get _propertyForValue() {
```

Comment:
Removing this protected API is kind of a breaking change for combo-box extensions.
Consider reverting this change, as we generally only need it in `combo-box-light`.

Reply:
Reverted.